app/src/main/java/com/kms/cura/utils/PatientProfileCurrent.java

```diff
@@ -0,0 +1,31 @@
+package com.kms.cura.utils;
+
+import com.kms.cura.entity.user.PatientUserEntity;
+
+/**
+ * Created by linhtnvo on 6/8/2016.
+ */
+public class PatientProfileCurrent {
```

Comment:
So this class only save current patient ? How about doctor or admin ? And how about when you need to check the current user type ?